app/assets/stylesheets/app.scss

```diff
@@ -19,6 +19,18 @@ html {
 
 body {
   margin: 0;
+  position: relative;
+  padding-bottom: 209px;
+  min-height: 100vh;
+}
+
+@media only screen and (max-width: 500px) {
+  body {
+    margin: 0;
+    position: relative;
+    padding-bottom: 545px;
+    min-height: 100vh;
+  }
```

Comment:
```css
margin: 0;
position: relative;
min-height: 100vh;
```
These rules are already there for body. No need of adding them here. Avoid duplication like this

Reply:
Yeah we don't need them. Have made the changes